github/src/main/kotlin/gropius/sync/github/model/TimelineItemDataCache.kt

```diff
@@ -8,6 +8,7 @@ import org.springframework.data.mongodb.core.mapping.Document
 
 /**
  * Cache for unprocessed timeline items
+ * @param imsProject IMSProject of the repo
```

Comment:
please be consistent with the way you document params in one class

Reply:
af217fde871ded5eb3e03a21b7d408b96b04189a